Maze/MazeController.swift

```diff
@@ -119,8 +120,10 @@ class MazeController {
         }
     }
     
-    func moveObjectToLocation(var object: MazeObject, newLocation: MazeLocation, completionHandler: () -> Void) {
+    func moveObjectToLocation(obj: MazeObject, newLocation: MazeLocation, completionHandler: () -> Void) {
 
+        var object = obj
```

Comment:
^ see above. Shouldn't this method belong to the MazeObject? Something like MazeObject.moveToLocation(newLocation: MazeLocation, completionHandler: () -> Void)?